app/services/data/submit-claim-response.js

```diff
@@ -78,6 +78,10 @@ function updateClaim (claimId, caseworker, decision, note, visitConfirmationChec
     }
   }
 
+  if (updateObject.Note.length > 250) {
+    updateObject.Note = updateObject.Note.substring(0, 250)
+  }
+
```

Comment:
I'm not sure that just truncating the field is the best idea - this gives the illusion that you can add more text than you really can (unless the text box is limited to only 250 characters)